exporter/opentelemetry-exporter-opencensus/tests/test_otcollector_trace_exporter.py

```diff
@@ -77,7 +77,7 @@ def test_proto_timestamp_from_time_ns(self):
 
     # pylint: disable=too-many-locals
     # pylint: disable=too-many-statements
-    def test_translate_to_collector(self):
+    def test__translate_to_collector(self):
```

Comment:
Nit
```suggestion
    def test_translate_to_collector(self):
```

Reply:
Well, this is testing `_test_translate_to_collector` :sweat_smile: